src/NServiceBus.AcceptanceTests/EndpointTemplates/TestSuiteConstraints.cs

```diff
@@ -9,6 +9,8 @@ public interface ITestSuiteConstraints
         bool SupportNativePubSub { get; }
 
         bool SupportNativeDeferral { get; }
+
+        bool ProvideOutboxPersistence { get; }
```

Comment:
I think `Supports` is probably better here. for example:

"The RabbitMQ transport supports native pubsub."